### ISSUE TITLE
Toast position top right has a very large top padding by default

### ISSUE THREAD

**Issue by dudelisius** [livewire/flux#387]
Hi there! 👋

With the new toast variants the top right toast is shown with a ver big top padding (6rem). This works when having a top navbar as the docs have. However I use the sidebar option only and the padding is random. 

In line with the "bring your own spacing" I would expect the toast to always have the same padding around it within the window.

Code to reproduce; 

`Flux::toast(heading: __('Changes saved!'), text: __('Successfully updated the team settings!'), variant: 'success', position: 'top right')`

Screenshot;

![Image](https://github.com/user-attachments/assets/aae9ab40-51ec-4f0b-b782-d9fd135ce806)

**Comment by calebporzio** [comment]
Good call. I just changed it so that you have to add it manually instead.

![Image](https://github.com/user-attachments/assets/0b7a5cbe-e545-4514-882b-11a7dc6d7928)
